### DISCUSSION TITLE
Yay I am a GitHub Global Campus Student!

### DISCUSSION THREAD

**Discussion by adityakanu** [skills/.github#129] [General]
Please check out my profile [github.com/adityakanu](@adityakanu) 
Would love to connect on LinkedIn as well: https://linkedin.com/in/adityakanu

**Comment by The11sun** [comment]
Best regards, github support team I had no intention of ignoring any
attention, but I personally didn't want to do it anymore..." I'm so tired
of the past time..." I blame myself for my own ability and knowledge... So
I did what I myself had seen the results and got it." I was worried that
the outcome would be the same.. I have to live with different factors every
day... I apologize for this opportunity... I was stressed when I couldn't
discern when so many groups of people surrounded me.

それは私自身のスティントです。

ในวันที่ ส. 26 พ.ย. 2022 14:48 น. Aditya Kanu ***@***.***>
เขียนว่า:
